src/Api/Response.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Jahuty\Api;
+
+class Response
```

Comment:
this should probably implement `Psr\Http\Message\ResponseInterface`

Reply:
Same as ☝️.